src/alert/alert.js

```diff
@@ -0,0 +1,17 @@
+import jQuery from 'jquery';
+import template from './alert.pug';
+
+export default function alert(message) {
+	const html = template({message : message});
```

Comment:
never use tabs for indentation, always use 2 spaces or whatever your team is using (but I'm sure it won't be tabs....)